lib/ads/client_side_ad_manager.js

```diff
@@ -191,15 +199,15 @@ shaka.ads.ClientSideAdManager = class {
       // because 'loadedmetadata' is sometimes called before the video resizes
       // on some platforms (e.g. Safari).
       this.eventManager_.listen(this.video_, 'loadeddata', () => {
-        const viewMode = document.fullscreenElement ?
+        const viewMode = isInFullscreen ?
```

Comment:
The fullscreen status might have changed before this event listener is called.
I'd make a method for determining the fullscreen status, instead of storing it as a variable.